src/games/strategy/triplea/TripleA.java

```diff
@@ -90,7 +92,7 @@ public void shutDown() {
 
 
   @Override
-  public void startGame(final IGame game, final Set<IGamePlayer> players, final boolean headless) {
+  public void startGame(final IGame game, final Set<IGamePlayer> players, final boolean headless, Runnable exceptionAction) {
```

Comment:
Thinking about this, having a return flag is pretty appropriate. There are a number of cases where a game will not start that are not exceptional. For example, simply not having a map.  A user can trigger that error repeatedly via normal UI controls, therefore it is not an exception since it can be user generated. Passing a runnable is effectively a callback, same thing with exceptions. The return flag is still the same, but the client decides what to do with the response  as part of an if/else block. The control flow is at least explicit.

Failing that, I would consider removing the thing that is causing 'startGame' to swallow exceptions.